my-app/src/app_context/song-context.jsx

```diff
@@ -0,0 +1,49 @@
+import { createContext, useState } from 'react';
+import { songCards } from '../data.js';
+import { v4 as uuidv4 } from 'uuid';
+
+export const SongContext = createContext({
+  songs: [],
+  count: 0,
+  getSelectedCards: () => {},
+  deleteSelectedCards: () => {},
+  addCard: () => {},
+});
+
+export default function SongContextProvider({ children }) {
+  const [songs, setSongs] = useState(
+    songCards.map(songCard => ({ ...songCard, isChecked: false }))
```

Comment:
To avoid unnecessary mapping you can add `isChecked: false` directly to the `data.js` file.